### DISCUSSION TITLE
Need information about dns

### DISCUSSION THREAD

**Discussion by iamxhunt3r** [projectdiscovery/nuclei-templates#11231] [General]
Hello ,

can i form up nuclei template with below dig flags

dig +trace +question +qr +comments +nodnssec -t NS -q $DOMAIN

Thanks.

**Comment by iamxhunt3r** [comment]
https://github.com/google/broken-dns

**Reply by iamxhunt3r** [reply]
@ehsandeep , Can you take a look.

Thanks

**Reply by princechaddha** [reply]
Hi @iamxhunt3r, the response time to this thread was longer than usual and it was somehow overlooked. Someone from the dev team will reply on this shortly.
